public/content/documentation/native/controls/dropdown.md

```diff
@@ -0,0 +1,337 @@
+
+   - WHEN I swipe to navigate to the dropdown 
+      - THEN the focus should move to the dropdown 
+         - AND the dropdown's name, role, value, and state (EXPANDED or COLLAPSED) should be expressed 
+   - WHEN I double-tap the dropdown 
+      - THEN the dropdown should be selected and opened
+
+3. Scenario: Test screen reader output on all devices
+
+   - WHEN a screen reader reads the dropdown 
+      - THEN its name should clearly describe its purpose and match any visible label 
+         - AND its role should be identified as a button in iOS and as "double tap to activate" in Android 
+         - AND its visible label should be grouped or associated with the dropdown in a single swipe 
+         - AND its state (DISABLED/DIMMED) should be expressed 
+         - AND the EXPANDED or COLLAPSED state should be announced on the elements that close or open the dropdown
+
+4. Scenario: Test device OS settings for text resize
+
+   - WHEN I adjust the device text resize setting to 200%
+      - THEN the text on the button should resize up to 200% without losing information
+
+Full information: [https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/dropdown](https://www.magentaa11y.com/MagentaA11yV2#/native-criteria/controls/dropdown)
+
+## iOS Developer Notes
+There is no native dropdown element for iOS.  The notes below are suggestions and accessibility guidance.
+
+### General Notes
+- A dropdown is a button that opens a list of options.  When an option is chosen, it displays in the field as the value
+- The difference between a menu and dropdown is a menu item performs an action when activated.  A dropdown item only replaces the current option
+- The screen reader focus moves directly to first option in dropdown upon double tapping the dropdown button and is confined in the list.
+- Sometimes, a hidden "dismiss context menu" button after the last item is available to close it.
+- Focus should go back to the triggering dropdown button, displaying the new option
+- There must be a visible label for the dropdown field that is not a placeholder and it describes the purpose of the dropdown.
+- The screen reader focus also remains confined in the dropdown list
+- The state of expanded or collapsed should be announced
+
+### Name
+- Programmatic name describes the purpose of the control.
+- If visible text label exists, the programmatic name should match the visible text label.
+    - **Note:** Setting a programmatic name while a visible text label exists may cause VoiceOver to duplicate the announcement of the name. If this happens, hide the visible text label from VoiceOver recognition
+- When naming a button, do not add "button" to the programmatic name (label). Assigning "Button" as the role will handle this announcement.
+  - **Incorrect announcement:** "Submit button, Button"
+  - **Correct announcement:** "Submit, Button"
+- Placeholder or value text is NOT the programmatic name
+
+**UIKit**
```

Comment:
Since "UIKit" and "SwiftUI" below aren't headings, and there are multiples under each "Name", "Role", "Groupings", "State", and "Focus", I think these should likely stay indented and bulleted as it is in current-day MagentaA11y.